CONTRIBUTING.md

```diff
@@ -1,11 +1,78 @@
-# Contributing
+# Overview
 
-**Technical Steering Committee (TSC)**
+This project aims to be governed in a transparent, accessible way for the benefit of the community. All participation in this project is open and not bound to corporate affilation. Participants all our bound to the ASWF Code of Conduct.
+
+# Project roles
+
+## Contributor
+
+The contributor role is the starting role for anyone participating in the project and wishing to contribute code.
+
+# Process for becoming a contributor
+
+* Review the [coding standards](http://www.openvdb.org/documentation/doxygen/codingStyle.html) to ensure your contribution is in line with the project's coding and styling guidelines.
+* Sign the Individual CLA and have your organization sign the Corporate CLA.
+* Submit your code as a PR with the appropriate DCO signoff.
+* Have your submission approved by the maintainer(s) and merged into the codebase.
+
+## Committer
+
+The committer role enables the participant to commit code directly to the repository, but also comes with the obligation to be a responsible leader in the community.
+
+### Process for becoming a committer
+
+* Show your experience with the codebase through contributions and engagement on the community channels.
+* Request to become a committer.
+* Have the majority of committers approve you becoming a committer.
+* Your name and email is added to the MAINTAINERS.md file for the project.
+
+### Committer responsibilities
+
+* Monitor email aliases.
+* Monitor Slack (delayed response is perfectly acceptable).
+* Triage GitHub issues and perform pull request reviews for other committers and the community.
+* Make sure that ongoing PRs are moving forward at the right pace or close them.
+* In general continue to be willing to spend at least 25% of ones time working on the project (~1.25 business days per week).
+
+### When does a committer lose committer status?
+
+If a committer is no longer interested or cannot perform the committer duties listed above, they
+should volunteer to be moved to emeritus status. In extreme cases this can also occur by a vote of
+the committers per the voting process below.
+
+## Technical Steering Committee (TSC) member
+
+The Technical Steering Committee (TSC) oversees the overall technical direction of OpenVDB, as defined in the [charter](charter.md).
```

Comment:
Perhaps worth mentioning that TSC members and committers/maintainers are the same individuals. Then we can consider re-visiting this if necessary in the future. Based on the size of the OpenVDB community, I doubt this will be necessary though.